src/core/thread/mle_router.cpp

```diff
@@ -2454,7 +2454,7 @@ ThreadError MleRouter::HandleNetworkDataUpdateRouter(void)
         {
             if (child->mNetworkDataVersion != mNetif.GetNetworkDataLeader().GetStableVersion())
             {
-                static const uint8_t responseTlvs[] = {Tlv::kNetworkData, Tlv::kActiveDataset, Tlv::kPendingDataset};
+                static const uint8_t responseTlvs[] = {Tlv::kNetworkData};
```

Comment:
If only including `Tlv::kNetworkData`, Line 2420 already defines `static const uint8_t tlvs[] = {Tlv::kNetworkData};`, not necessary to define the new `responseTlvs` here?